src/mono/browser/runtime/loader/assets.ts

```diff
@@ -418,7 +421,7 @@ export function prepareAssets () {
                         behavior: "icu",
                         loadRemote: true
                     });
-                } else if (name === "segmentation-rules.json") {
+                } else if (name.startsWith("segmentation-rules") && name.endsWith(".json")) {
```

Comment:
curious why we don't use behavior here

Reply:
This belongs to the SWA PR, any we are using `behavior: "segmentation-rules"` like below, but this is reading from boot.json where it's part of `resources.icu`. I don't know why it's not a separate group.